sample_cdr_georef/sample_cdr_georef/server.py

```diff
@@ -237,6 +237,92 @@ async def georeference_map(req: MapEventPayload,  response_model=GeoreferenceRes
     return results
 
 
+def dummy_feature_results():
```

Comment:
Does this result 'make sense' for someone looking at it? I.e., this dummy result doesn't supply map_unit information so '11' doesn't mean anything.

Reply:
I have now replaced this with the object that is expected in the FeatureResults schema.